lib/sycamore/sycamore/query/planner.py

```diff
@@ -379,7 +388,7 @@ class LlmPlanner:
     def __init__(
         self,
         index: str,
-        data_schema: OpenSearchSchema,
+        data_schema: typing.Union[OpenSearchSchema, Schema],
```

Comment:
not a big deal at all, but why not just import Union -- looks like you already import a bunch of things from typing.

Reply:
oops IDE didn't want to. Will fix